iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/IIOPRootDefinition.java

```diff
@@ -72,12 +72,12 @@
 
     protected static final AttributeDefinition SOCKET_BINDING = new SimpleAttributeDefinitionBuilder(
             Constants.ORB_SOCKET_BINDING, ModelType.STRING, true).setAttributeGroup(Constants.ORB)
-            .setDefaultValue(new ModelNode().set("iiop")).setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
```

Comment:
If the intent is to make it required, we need a version bump since this has been in 9